File: src/main/java/org/polyfrost/polyblur/commands/PolyBlurCommand.java
package org.polyfrost.polyblur.commands;

import org.polyfrost.oneconfig.api.commands.v1.factories.annotated.Command;
import org.polyfrost.polyblur.PolyBlur;
import org.polyfrost.utils.v1.dsl.ScreensKt;

@Command("polyblur")
public class PolyBlurCommand {

    @Command
    public void main() {
        ScreensKt.openUI(PolyBlur.instance.config);
    }

}

File: src/main/kotlin/org/polyfrost/polyblur/PolyBlurCommand.kt
package org.polyfrost.polyblur

import org.polyfrost.oneconfig.api.commands.v1.factories.annotated.Command
import org.polyfrost.oneconfig.utils.v1.dsl.openUI

@Command(PolyBlur.ID)
class PolyBlurCommand {

    @Command
    fun main() {
        PolyBlurConfig.openUI()
    }

}
